### PR TITLE
Refactor Documentation API Horus

### PR DESCRIPTION
**Pull request by RamonHossein**
I organized the codes in tags according your programming languages (shell, cli).
- CLI is tagged how ruby because the Api of documentation only accept programming languages already predefined.
- I only changed the structure of text, no your contents.

### CODE REVIEW

**Comment by RamonHossein** [comment]
Please, attention in my grammar. I not write in English very well.

**Comment by fernandes** [comment]
@rebyoliveira please!

File: source/index.md [hunk 1]
```diff
@@ -13,38 +14,46 @@ search: true
...
 ## Reseller (Admin)
 
- Reseller is the responsible for sell products and services Softlayer (initially) to Clients.
+Reseller is the responsible for sell products and services Softlayer (initially) to Clients.
 
- Authentication as Reseller allows the user to list and update information about yourself account.
+Authentication as Reseller allows the user to list and update information about yourself account.
 
- Authentication as Reseler allows the user to list, create and update informations about Users.
+Authentication as Reseler allows the user to list, create and update informations about Users.
```
Comment (rebyoliveira) [inline source/index.md]: Correct: Reseller.

File: source/index.md [hunk 2]
```diff
@@ -52,162 +61,1289 @@ The Horus API does a division of authentication according to the type of user, W
...
+      }
+    }
+  }
+}
+```
+
+### List
+
+You also can list your resources using a command line or accessing a route, both cases are described below. In this exeample we use the manager.
+
```
Comment (rebyoliveira) [inline source/index.md:440]: Correct: "example" not exeample:  You also can list your resources using a command line or accessing a route, both cases are described below. In this example we use the manager.

File: source/index.md [hunk 3]
```diff
@@ -52,162 +61,1289 @@ The Horus API does a division of authentication according to the type of user, W
...
+      }
+    }
+  }
+}
+```
+
+### Update
+
+You also can update a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this exeample we use the manager.
+
```
Comment (rebyoliveira) [inline source/index.md:353]: Correct: "example" not 'exeample' : You also can update a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this example we use the manager.

File: source/index.md [hunk 4]
```diff
@@ -52,162 +61,1289 @@ The Horus API does a division of authentication according to the type of user, W
...
+      "last-name":"Manager"
+    }
+  }
+}
+```
+
+### Update
+
+You also can updatet your profile using a command line or accessing a route, both cases are described below. In this exeample we use the manager.
+
```
Comment (rebyoliveira) [inline source/index.md:204]: Correct: "example" not exeample:

File: source/index.md [hunk 5]
```diff
@@ -52,162 +61,1289 @@ The Horus API does a division of authentication according to the type of user, W
...
+        }
+      }
+    }
+  ]
+}
+```
+### Show
+
+You also can show a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this exeample we use the manager.
+
```
Comment (rebyoliveira) [inline source/index.md:568]: Correct: "example" not exeample

**Comment by rebyoliveira** [comment]
Finished review. Just need to correct the words above.

**Comment by RamonHossein** [comment]
Correction of words above finished.

**Pull request merged**